docs/develop/ruby/index-ruby.mdx

```diff
@@ -24,31 +24,57 @@ Additional Ruby clients for Redis can be found under the Ruby section of the Red
   ]}>
   <TabItem value="redis-rb">
 
+Redis is an open source, in-memory, key-value data store most commonly used as a primary database, cache, message broker, and queue. Redis cache delivers sub-millisecond response times, enabling fast and powerful real-time applications in industries such as gaming, fintech, ad-tech, social media, healthcare, and IoT.
 
+#### Step 1. Run a Redis server
 
-### Step 1. Clone the repository
+You can either run Redis in a Docker container or directly on your Mac OS.
+Follow the below commands to setup a Redis server locally:
+
+
+```
+ brew tap redis-stack/redis-stack
+ brew install --cask redis-stack
+```
+
+:::info INFO
+Redis Stack unifies and simplifies the developer experience of the leading Redis modules and the capabilities they provide. Redis Stack bundles five Redis modules: RedisJSON, RedisSearch, RedisGraph, RedisTimeSeries, and RedisBloom.
+[Learn more](/create/redis-stack)
+:::
+
+
+Ensure that you are able to use the following Redis command to connect to the Redis instance.
+
+```bash
+ redis-cli -h localhost -p 6379
+ localhost>
+```
+
+Now you should be able to perform CRUD operation with Redis commands. For example, you can insert data to Redis via SET command and the fetch it via GET command. The above redis client command might require password if you have setup authentication in your redis configuration file. If Redis password is not set, then you can connect to Redis instance flawlessly. Be default, Redis uses the port 6379 that can be modified in the redis configuration file.
```

Comment:
```suggestion
Now you should be able to perform CRUD operations with Redis commands. For example, you can insert data to Redis with the `SET` command and the fetch it with the `GET` command. The above Redis client command might require a password if you have setup authentication in your Redis configuration file. By default, Redis listens on port 6379.  This can be modified in the Redis configuration file.
```